tools/template_based_query_generation/src/main/resources/user_config/dql.json

```diff
@@ -0,0 +1,49 @@
+{
+<<<<<<< HEAD
```

Comment:
Please resolve those merging conflicts

Reply:
Ahh, sorry I missed the test folder and config files. Just patched this part.